UnitySDK/UnitySDK.sln.DotSettings

```diff
@@ -1,6 +1,17 @@
 ﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+	<s:Boolean x:Key="/Default/UserDictionary/Words/=Blas/@EntryIndexedValue">True</s:Boolean>
```

Comment:
Not sure if you can do this, but I think it should be all caps (BLAS = Basic Linear Algebra Subprograms).  Maybe add CPU and GPU too?

Reply:
sounds good